packages/p2p/src/pages/buy-sell/buy-sell-form.jsx

```diff
@@ -49,9 +42,22 @@ const BuySellForm = props => {
 
     const exchange_rate = useP2PExchangeRate(local_currency);
     const order_completion_time = order_expiry_period / 60;
-
-    const [previous_rate, setPreviousRate] = React.useState(exchange_rate);
     const [input_amount, setInputAmount] = React.useState(min_order_amount_limit);
+    const [current_effective_rate, setCurrentEffectiveRate] = React.useState(0);
+    const [changed_rate, setChangedRate] = React.useState(undefined);
```

Comment:
shouldn't the initialvalue be 0 or ''?

Reply:
i intentionally left it as undefined because of the line 127

`const default_effective_rate = changed_rate ?? effective_rate;` 

cause i am checking for undefined, though i could leave it as null, but i think undefined is ok. If i did 0 or '' i have to intentionally check for 0 or '' like so

`const default_effective_rate = changed_rate !== 0 ? changed_rate : effective_rate;`
`const default_effective_rate = changed_rate !== '' ? changed_rate : effective_rate;`